portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +35,69 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
```

Comment:
Move to a different endpoint. That keeps the data endpoint for just fetching the comments.

Reply:
Will be addressed when I refactor code as an update to PR #14

Reply:
Done 265b9c0